packs/data/pfs-season-4-bestiary.db/overtaxed-vrisk.json

```diff
@@ -1741,7 +1741,7 @@
             "type": "lore"
         }
     ],
-    "name": "Overworked Vrisk",
+    "name": "Overtaxed Vrisk",
```

Comment:
If Deidril's supports this scenario we may need to give them a heads up about the name change if they reference with names.